CHANGELOG.md

```diff
@@ -40,6 +40,7 @@
 - Minor: Don't show update button for nightly builds on macOS and Linux, this was already the case for Windows (#2163, #2164)
 - Minor: Tab and split titles now use display/localized channel names (#2189)
 - Minor: Add a setting to limit the amount of historical messages loaded from the Recent Messages API (#2250, #2252)
+- Minor: Color mentions to match the mentioned users (#1963, #2121, #2284)
```

Comment:
We don't include issue numbers in changelog entries. Pharsing seems a bit weird to me, I hope this makes sense.
```suggestion
- Minor: Added a setting for @mentions to match the mentioned users' chat color (#2284)
```

Reply:
This not necessarily true. (For example, we also link #2163 or #2250)

Reply:
We do a bit of both - this is fine imo.

I basically see it as a list of relevant PRs or Issues to the change.